crates/bevy_sprite/src/bundle.rs

```diff
@@ -30,6 +32,8 @@ pub struct SpriteBundle {
 pub struct SpriteSheetBundle {
     /// The specific sprite from the texture atlas to be drawn, defaulting to the sprite at index 0.
     pub sprite: TextureAtlasSprite,
+    /// Defines if how the sprite behaves along its size
```

Comment:
/// Controls how the image is altered when scaled.

Reply:
```suggestion
   /// Controls how the image is altered when scaled.
```